File: Examples/DataManagementPreferenceActivity/src/course/examples/DataManagement/PreferenceFragment/PreferencesActivityExample.java
package course.examples.DataManagement.PreferenceFragment;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.view.View.OnClickListener;
import android.widget.Button;

public class PreferencesActivityExample extends Activity {

	@Override
	public void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.main);

		// Open a User Preferences Entry Activity 
		final Button button = (Button) findViewById(R.id.check_pref_button);
		button.setOnClickListener(new OnClickListener() {
			@Override
			public void onClick(View v) {
				startActivity(new Intent(
						PreferencesActivityExample.this,
						ViewAndUpdatePreferencesActivity.class));
			}
		});

	}
}

File: ExamplesKotlin/DataManagementPreferenceFragment/app/src/main/java/course/examples/datamanagement/preferencefragment/PreferencesActivityExample.kt
package course.examples.datamanagement.preferencefragment

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View

class PreferencesActivityExample : AppCompatActivity() {

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.main)
    }

    // Open a User Preferences Entry Activity
    fun onButtonClick(v: View) {
        startActivity(
            Intent(
                this@PreferencesActivityExample,
                ViewAndUpdatePreferencesActivity::class.java
            )
        )
    }
}